.travis.yml

```diff
@@ -26,13 +31,15 @@ before_install:
 
 install:
     - pip install mkdocs mkdocs-material
+    - curl -sL -o ${HADOLINT} "https://github.com/hadolint/hadolint/releases/download/v1.16.0/hadolint-$(uname -s)-$(uname -m)" && chmod 700 ${HADOLINT}
 
 script:
   - docker build --target=builder --tag=doccano-test .
   - >
     if [[ "${DATABASE}" != "sqlite" ]]; then
       docker run --network doccano -e DATABASE_URL="${DATABASE_URL}" -it doccano-test sh -c 'app/manage.py wait_for_db && app/manage.py migrate && app/manage.py test api.tests server.tests'
     fi
+  - git ls-files --exclude='Dockerfile*' --ignored | xargs --max-lines=1 ${HADOLINT}
```

Comment:
Given that we only have a single Dockerfile in the project, why not just `hadolint Dockerfile`?